addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -210,6 +210,11 @@ def full_filter_invl_to_apply(invoice_line):
             tax_details_info['NoSujeta']['ImportePorArticulos7_14_Otros'] = round(sign * base_amount_not_subject, 2)
         if not invoice.company_id.currency_id.is_zero(base_amount_not_subject_loc) and invoice.is_sale_document():
             tax_details_info['NoSujeta']['ImporteTAIReglasLocalizacion'] = round(sign * base_amount_not_subject_loc, 2)
+        if not tax_details_info and invoice.is_sale_document():
+            if any(t['l10n_es_type'] == 'no_sujeto_loc' for t in tax_details['tax_details'].values()):
+                tax_details_info['NoSujeta']['ImportePorArticulos7_14_Otros'] = 0
```

Comment:
based on the four lines above (209-212), shouldn't this be the opposite? "ImportePorArticulos7_14_Otros" should be set for 0 line with no_sujeto and ImporteTAIReglasLocalizacion fro 0 line with no_sujeto_loc

Reply:
Yes, you are probably right